dm/worker/config.go

```diff
@@ -266,26 +312,43 @@ func (c *Config) adjustFlavor() error {
 			return terror.ErrNotSupportedFlavor.Generate(c.Flavor)
 		}
 	}
-	// decrypt password
-	clone, err := c.DecryptPassword()
-	if err != nil {
-		return err
+
+	c.Flavor, err = utils.GetFlavor(ctx, db)
+	if ctx.Err() != nil {
+		err = terror.Annotatef(err, "time cost to get flavor info exceeds %s", dbGetTimeout)
+	}
+	return terror.WithScope(err, terror.ScopeUpstream)
+}
+
+func (c *Config) adjustServerID(ctx context.Context, db *sql.DB) error {
+	if c.ServerID != 0 {
+		if c.ServerID < 0 || c.ServerID > maxServerID {
```

Comment:
do we need to change the type of `serverID` (`int`) or update the value of `maxServerID` (max value of `uint32`)?

Reply:
update https://github.com/pingcap/dm/pull/329/commits/457bbbff705e425ba084eee8a5e5deb959eaa8da